src/components/controls/AlertNotice.svelte

```diff
@@ -0,0 +1,73 @@
+<script>
+  import { fade } from 'svelte/transition';
+  import { cubicOut } from 'svelte/easing';
+
+  // So we don't pollute the entire LS namespace.
+  const KEY_PREFIX = 'alertNotice-';
+  export let dismissText = 'dismiss'; // Label of 'dismiss' button.
+
+  // Required: Which localStorage key to check whether this notice has been dismissed.
+  export let toggleKey = '';
+
+  $: alertVisible = window.localStorage.getItem(KEY_PREFIX + toggleKey) !== 'true';
+
+  function dismissNotice() {
+    alertVisible = false;
+    window.localStorage.setItem(KEY_PREFIX + toggleKey, 'true');
+  }
+</script>
+
+<style>
+  .alert-notice {
+    position: absolute;
+    left: var(--space-4x);
```

Comment:
For some reason, it feels more natural to me on the right. I want to say it's because of some common UI convention or something, but it may just be personal preference.